app/src/main/java/ch/epfl/sdp/drone3d/mission/ParallelogramMappingMissionService.kt

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2021  Drone3D-Team
+ * The license can be found in LICENSE at root of the repository
+ */
+
+package ch.epfl.sdp.drone3d.mission
+
+import ch.epfl.sdp.drone3d.drone.DroneService
+import com.mapbox.mapboxsdk.geometry.LatLng
+import java.lang.IllegalArgumentException
+import javax.inject.Inject
+
+class ParallelogramMappingMissionService @Inject constructor(override val droneService: DroneService):MappingMissionService{
```

Comment:
The style is not perfect, there should be a space after the :

Reply:
Corrected